src/main/java/com/checkmarx/flow/custom/GitLabIssueTracker.java

```diff
@@ -263,7 +264,11 @@ private String getFileUrl(ScanRequest request, String filename) {
             return null;
         }
         String repoUrl = request.getRepoUrl().replace(".git", "/");
-        return repoUrl.concat("/blob/").concat(request.getBranch()).concat("/").concat(filename);
+        if (Optional.ofNullable(filename).isPresent()) {
```

Comment:
the if and the else are for return.
so it will be better to write it as:
```Java
return (Optional.ofNullable(filename).isPresent()) 
  ? <the string to return> 
  : null;
```
or
```Java
return (filename == null) 
  ? null 
  : <the string to return>;
```

Reply:
Done.